### ISSUE TITLE
"Bad file descriptor" on FC5 x86 during "Configure Selected OSCAR Packages" step

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#31]
**Reported by bli on 28 May 2006 19:41 UTC**
Tested with trunk r4878, I get the following during Step 2:

```
=============================================================================
== Running step 2 of the OSCAR wizard: Configure selected OSCAR packages
=============================================================================

--> About to run /opt/oscar-5.0a1r4878M/packages/kernel_picker/scripts/pre_configure for kernel_picker
Bad file descriptor at /usr/lib/perl5/5.8.8/File/Find.pm line 822
Bad file descriptor at /usr/lib/perl5/5.8.8/File/Find.pm line 822
Bad file descriptor at /usr/lib/perl5/5.8.8/File/Find.pm line 822
update-rpms: cache not found.
--> About to run /opt/oscar-5.0a1r4878M/packages/switcher/scripts/pre_configure for switcher
```

Not sure where the error message is coming from, but could be ```kernel_picker``` or ```update-rpms```...

**Comment by dikim33** [comment]
**Comment by bli on 19 Jul 2006 04:04 UTC**
Error message disappeared after kernel_picker package is removed.  Will close after kernel_picker is removed from branch-5-0.

**Comment by dikim33** [comment]
**Comment by bli on 20 Jul 2006 04:57 UTC**
(In [5196]) kernel_picker is not essential for OSCAR 5.0 since Erich made changes to SystemInstaller/Systemconfigurator
to support SSI-OSCAR.  Development for kernel_picker can continue in trunk if necessary.  This closes #31.
